docs/designers-developers/developers/backward-compatibility/README.md

```diff
@@ -1 +1,65 @@
 # Backward Compatibility
+
+Historically WordPress has been known for preserving backward compatibility across versions. Gutenberg follows this example wherever possible in its production public APIs. There are rare occasions where breaking backward compatibility is unavoidable and in those cases the breakage:
+
+* should be constrained as much as possible to a small surface area of the API.
+* should be documented as clearly as possible to third-party developers using dev notes.
+
+## What qualifies as a production public API
```

Comment:
```suggestion
## What qualifies as a production public API?
```

Reply:
If we do that, we should do it for all the titles that are written as questions.

Reply:
The "Hows" read more like statements.

Although technically, this can also be a statement, it reads more like a question, but it might be just me.

Reply:
It's a content clause, it's perfectly fine as is.